Modules/QR Reader/MITScannerMgr.m

```diff
@@ -86,7 +88,9 @@ - (void)setupCaptureSession
     self.deviceInput = [[AVCaptureDeviceInput alloc] initWithDevice:self.captureDevice error:&error];
     if( self.deviceInput == nil )
```

Comment:
@bskinner Hmm.. I'm making a change now as you suggested, however, I did verify the behavior of checking deviceInput for nil. The device will in fact be nil if access wasn't granted by the user. Here's what documentation says for the return value of `initWithDevice`: 

`An AVCaptureDeviceInput instance that provides data from the given device, or nil, if the device could not be used for capture.`

Additional info from the documentation:

`If the device cannot be opened because it is no longer available or because it is in use, for example, this method returns nil, and the optional outError parameter points to an NSError describing the problem.`

So in case user is not authorized, then deviceInput will be nil and error would say something like that: `"Back Camera is not authorized"`